tuna/rocmlir/rocmlir_tables.py

```diff
@@ -215,7 +277,6 @@ def session(self):
     """session column"""
     return Column(Integer, ForeignKey("session_rocmlir.id"), nullable=False)
 
-  config = Column(Integer, ForeignKey("rocmlir_conv_config.id"), nullable=False)
```

Comment:
Moving away from foreign-key config IDs and towards local config strings.  We don't really need the ID as such, and as a foreign key it can prevent deleting a config without deleting results.  We could easily want to flush and reload configs while preserving results for the history.  Another thought is to add a session ID to the config tables.